source/virtualBuffers/__init__.py

```diff
@@ -539,6 +539,13 @@ def script_refreshBuffer(self,gesture):
 	# Translators: the description for the refreshBuffer script on virtualBuffers.
 	script_refreshBuffer.__doc__ = _("Refreshes the document content")
 
+	@script(
+		# Translators: the description for the toggleScreenLayout script on virtualBuffers.
+		description=_(
+			"Toggles on and off if the screen layout is preserved while rendering the document content"
+		),
+		gesture="kb:NVDA+v",

```

Comment:
I believe this should also be removed from the `_gestures` at the end of the file.